storage_service/pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+django_find_project=false
+python_paths=/usr/lib/archivematica/storage-service/
+DJANGO_SETTINGS_MODULE=storage_service.settings.local
```

Comment:
I tried setting this to ```storage_service.settings.test```, but ran into http://stackoverflow.com/questions/19958587/django-test-in-memory-sqllite-missing-tables-during-test-run because of our startup() code in urls.py being run on import.  Django 1.7 should fix that, and then we can switch this to using the test settings.